packages/server/src/integrations/dynamodb.ts

```diff
@@ -131,7 +131,9 @@ module DynamoModule {
 
     constructor(config: DynamoDBConfig) {
       this.config = config
-      this.connect()
+      if (!this.config.endpoint) {
+        this.connect()
```

Comment:
Do we not need to call the connect if there is an `endpoint`?
How does that work?

Reply:
Nope, it doesn't need the AWS configuration and passes the endpoint straight into the `DynamoClient`